packages/@webex/internal-plugin-metrics/src/new-metrics.ts

```diff
@@ -54,6 +52,8 @@ class Metrics extends WebexPlugin {
     this.webex.once('ready', () => {
       // @ts-ignore
       this.callDiagnosticMetrics = new CallDiagnosticMetrics({}, {parent: this.webex});
+      // @ts-ignore
+      this.callDiagnosticLatencies = new CallDiagnosticLatencies({webex: this.webex});
```

Comment:
seeing that the constructor for `CallDiagnosticMetrics` takes webex instance as a `parent` and CallDiagnosticLatencies takes it as `webex` makes me wonder if maybe CallDiagnosticLatencies class should also extend a `WebexPlugin`?

Reply:
i've refactored to extend the plugin